src/pages/saksbehandling/steg/beregning/DelerAvPeriodeInputs.tsx

```diff
@@ -0,0 +1,71 @@
+import { FormikErrors } from 'formik';
+import { Normaltekst, Feilmelding } from 'nav-frontend-typografi';
+import React from 'react';
+import DatePicker from 'react-datepicker';
+import { IntlShape } from 'react-intl';
+
+import { FradragFormData } from '~pages/saksbehandling/steg/beregning/FradragInputs';
+import { DelerAvPeriode } from '~types/Fradrag';
+
+import styles from './fradragInputs.module.less';
+
+const DelerAvPeriodeInputs = (props: {
+    fraOgMedId: string;
+    tilOgMedId: string;
+    fradrag: FradragFormData;
+    setFieldValue: (field: string, value: Date | [Date, Date] | null) => void;
+    delerAvPeriodeErrors: FormikErrors<DelerAvPeriode> | undefined;
+    intl: IntlShape;
+}) => {
+    const fraOgMedError = props.delerAvPeriodeErrors?.fraOgMed;
+    const tilOgMedError = props.delerAvPeriodeErrors?.tilOgMed;
+
+    return (
+        <div className={styles.delerAvPeriodeContainer}>
+            <div>
+                <Normaltekst className={styles.dateLabelTekst}>
+                    {props.intl.formatMessage({ id: 'datovelger.fom.legend' })}
+                </Normaltekst>
+                <DatePicker
+                    id={props.fraOgMedId}
+                    name={props.fraOgMedId}
+                    selected={props.fradrag.delerAvPeriode?.fraOgMed}
+                    onChange={(e) => {
+                        props.setFieldValue(props.fraOgMedId, e);
+                    }}
+                    dateFormat="MM/yyyy"
+                    showMonthYearPicker
+                    isClearable
+                    selectsEnd
+                    startDate={props.fradrag.delerAvPeriode?.fraOgMed}
+                    endDate={props.fradrag.delerAvPeriode?.tilOgMed}
+                    minDate={props.fradrag.delerAvPeriode?.fraOgMed}
+                />
+                <Feilmelding>{fraOgMedError ?? ''}</Feilmelding>
```

Comment:
Finns det alltid en feilmelding?

Reply:
Nei, bare når det skjer feil med validering. default case er for å ikke å feilmelding på console.